lib/arc.d.ts

```diff
@@ -507,5 +511,9 @@ declare module "daostack-arc-js" {
       opts: ProposeContributionParams
     ): Promise<TransactionReceiptTruffle>;
     setParams(params: ContributionRewardParams): Promise<string>;
+
```

Comment:
I'm not sure these events should be part of this PR, better to be part of a broader task aimed at adding better support for events across all of the wrappers.  In any case, I don't see that they've been implemented in the ContributionReward class.

Reply:
thats true, and this is a good opportunity to do that work, but I needed these definitions to get my old alchemy demo working again. these functions come directly from the truffle contract

Reply:
are you ok with leaving these in for now while we work out how we want to handle the event functionality in the wrappers? in fact we may decide to just leave as is and pass through to the truffle contract handling for now

Reply:
sure